http-types-schema.json

```diff
@@ -86,6 +92,12 @@
                 { "type": "array", "items": { "type": "string" } }
               ]
           }
+        },
+        "timestamp": {
+          "title": "HttpResponseTimestamp",
+          "description": "The time at which the HTTP response was completed on the server side in ISO 8601 format, as in '2018-11-13T20:20:39+00:00'.",
+          "type": "string",
+          "format": "date-time"
```

Comment:
I assume it's never possible in practice to have both request and response timestamps in the object? Just wondering if we could simplify things by just having a single timestamp...

Reply:
I think what we ourselves need and get here (from things like server logs) is the response timestamp, when the response was done.

I've added the request timestamp, when the request was initiated, to (1) be able to measure time taken and (2) we might want to represent requests that caused a timeout (no response received). But if there is no immediate need for it I have no problem removing `request.timestamp` and revisiting it once needed - what do you think?

Reply:
We're adding both fields for now.